ReleaseNotes.md

```diff
@@ -1,3 +1,6 @@
+### New in 2.0.2

```

Comment:
I'd say that's a pretty significant new feature, so it deserves 2.1.0 version

Reply:
How does that exactly work? In the code I saw that git version is used to generate the AssemblyInfo content.

Reply:
I'll put a tag "2.1.0" on a git commit that will be a version realesed. And this file is edited by hand - so the numbers will have to coordinate